app/src/main/java/io/intrepid/russell/checkin/CheckInService.java

```diff
@@ -160,33 +167,32 @@ public void onResult(@NonNull Status status) {
     }
 
     private GeofencingRequest getGeofencingRequest() {
+        Geofence.Builder fenceBuilder = new Geofence.Builder()
+                .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_EXIT | Geofence.GEOFENCE_TRANSITION_DWELL)
+                .setExpirationDuration(Geofence.NEVER_EXPIRE)
+                .setLoiteringDelay(LOITERING_DELAY);
+
         return new GeofencingRequest.Builder()
                 .setInitialTrigger(GeofencingRequest.INITIAL_TRIGGER_DWELL | GeofencingRequest.INITIAL_TRIGGER_ENTER)
-                .addGeofence(new Geofence.Builder()
+                .addGeofence(fenceBuilder
                         .setRequestId(LocationData.THIRD_ST.id)
                         .setCircularRegion(LocationData.THIRD_ST.lat, LocationData.THIRD_ST.lng, LocationData.GEOFENCE_RADIUS)
-                        .setTransitionTypes(Geofence.GEOFENCE_TRANSITION_EXIT | Geofence.GEOFENCE_TRANSITION_DWELL)
-                        .setExpirationDuration(Geofence.NEVER_EXPIRE)
-                        .setLoiteringDelay(LOITERING_DELAY)
                         .build())
-                .addGeofence(new Geofence.Builder()
+                .addGeofence(fenceBuilder
                         .setRequestId(LocationData.ROGERS_ST.id)
                         .setCircularRegion(LocationData.ROGERS_ST.lat, LocationData.ROGERS_ST.lng, LocationData.GEOFENCE_RADIUS)
```

Comment:
For what it is, I'd say this is fine.  It's right on the edge of the kind of thing I'd refactor to iterate over a datastructure of locations.  The usual rule is "hard code it twice, third time refactor", though I've been known to refactor at the second time...